addons/adminMenu/functions/fnc_endMission.sqf

```diff
@@ -4,10 +4,11 @@ params ["_winningSide"];
 TRACE_1("params",_this);
 
 if (isServer) then {
-    private _missionType = getMissionConfigValue[QEGVAR(missionTesting,missionType), "other"];
-    [_winningSide, "Mission ended", _missionType] call ocap_fnc_exportData;
+    private _missionType = getMissionConfigValue[QEGVAR(missionTesting,missionType), 0];
+    private _missionTypeStr = ["other", "COOP", "TvT"] select _missionType;
```

Comment:
Are these also going to show up in the OCAP front-end as mission tags? If so, can "other" be capitalized?

Reply:
![image](https://github.com/BourbonWarfare/POTATO/assets/39413400/4a9c62c8-2cb1-470f-8359-e2d2fa732d64)
ex.